src/test/java/com/suse/salt/netapi/examples/Async.java

```diff
@@ -15,13 +18,14 @@
     private static final String PASSWORD = "saltdev";
 
     public static void main(String[] args) {
+        CloseableHttpAsyncClient httpClient = TestUtils.defaultClient();
```

Comment:
If the `defaultClient()` is something that could be used in *production* I would suggest to move it away from `TestUtils` to a better place.

Reply:
i don't think its suitable for production since its disables all timeout which you probably don't want in a production environment.

Reply:
Can't or shouldn't we provide some reasonable defaults for those timeouts, for example the values that were used before? I think in order to make it easy for people to get started quickly we could provide reasonable defaults with this default client.

Reply:
i added a helper method for creating an async http client with the default timeout values used in our config before

Reply:
Nice, so should we use `HttpClientUtils.defaultClient()` here? Do we still need to have `TestUtils.defaultClient()`?